threescale_api/resources.py

```diff
@@ -608,6 +608,7 @@ def promote(self, **kwargs) -> 'Proxy':
 class Service(DefaultResource):
     AUTH_USER_KEY = "1"
     AUTH_APP_ID_KEY = "2"
+    AUTH_BASIC = "3"
```

Comment:
Actually, auth mode `3` doesn't exist in apicast so we've got to think in something different..

Reply:
I need to admit I didn't check expected behavior for Basic auth using user_key, so it may differ from the "expectation" while that expectation follows behavior of behavior of auth in query.
@jsmadis HELP! to understand this

Reply:
I am a little bit lost in what are you trying to say. From that what I can understand I can say this:

- `auth mode 3` doesn't exist. There are 3 auth modes ( `1` - user key, `2` - app id & app key, `oidc` - OpenID connect). 

- auth_user_key is the name of the parameter that acts on behalf of user key ( You can change it in the integration page, authentication settings.) For example if you change it to `foo` (auth_user_key = foo) you will need to change requesting URL from
`https://<apicast_url>/?user_key=user_key`
to
`https://<apicast_url>/?foo=user_key`